src/test/java/webdriver/DriverManager.java

```diff
@@ -0,0 +1,21 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+
+public class DriverManager {
+    private static WebDriver driver = null;
+
+    public static WebDriver getDriver(){
+        return driver;
```

Comment:
if you call this  before setting driver it will cause you lots of NullPointerException, you need to safe such situations